x/community/client/proposal_handler.go

```diff
@@ -11,16 +11,22 @@ import (
 	"github.com/kava-labs/kava/x/community/types"
 )
 
-// community-pool deposit/withdraw lend proposal handlers
+// community-pool proposal handlers
 var (
+	// Deprecated: Use CommunityPoolProposal instead
 	LendDepositProposalHandler = govclient.NewProposalHandler(
 		cli.NewCmdSubmitCommunityPoolLendDepositProposal,
 		notImplementedRestHandler(types.ProposalTypeCommunityPoolLendDeposit),
 	)
+	// Deprecated: Use CommunityPoolProposal instead
 	LendWithdrawProposalHandler = govclient.NewProposalHandler(
 		cli.NewCmdSubmitCommunityPoolLendWithdrawProposal,
 		notImplementedRestHandler(types.ProposalTypeCommunityPoolLendDeposit),
 	)
+	ProposalHandler = govclient.NewProposalHandler(
+		cli.GetCmdSubmitCommunityPoolProposal,
+		notImplementedRestHandler(types.ProposalTypeCommunityPool),
```

Comment:
since there is a cli command for this, I assume this is implemented in the json RPC endpoint? Is there a reason why this isn't implemented in the rest or other non rpc transport apis (e.g. websocket or grpc), and where do we track / document which methods are implemented for a given transport?

Reply:
this is not grpc but legacy rest, which is now deprecated.